### PR TITLE
chore(sassdoc-theme): localize sassdoc shell strings

### LINKED ISSUE #3209: Localize SASSDOC shell strings
### Description  
Localize SASSDOC shell strings

- [ ] Defined in
- [ ] Parameters
- [ ] Name
- [ ] Description
- [ ] Type
- [ ] Default Value
- [ ] Authors
- [ ] Example

### PR DESCRIPTION
**Pull request by Aleksandyr**
Closes #3209

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [x] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by zdrawku** [comment]
@valadzhov, could you translate the following strings in the PR:
"Name": "Name",
        "Description": "Description",
        "Type": "Type",
        "Default Value": "Default Value",
        "Authors": "Authors",
        "Example": "Example",
        "Used by": "Used by"

You can propose them as Suggested change and Alex will approve them, this will spear you some time to get the branch and commit the changes ;)

File: extras/template/strings/shell-strings.json [hunk 1]
```diff
@@ -23,6 +23,13 @@
         "Enumeration": "列挙",
         "Static": "スタティック",
         "Components": "コンポーネント",
-        "Get Started": "はじめに"
+        "Get Started": "はじめに",
+        "Name": "Name",
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Name": "名",
```

File: extras/template/strings/shell-strings.json [hunk 2]
```diff
@@ -23,6 +23,13 @@
...
+        "Description": "Description",
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Description": "説明",
```

File: extras/template/strings/shell-strings.json [hunk 3]
```diff
@@ -23,6 +23,13 @@
...
+        "Type": "Type",
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Type": "型",
```

File: extras/template/strings/shell-strings.json [hunk 4]
```diff
@@ -23,6 +23,13 @@
...
+        "Default Value": "Default Value",
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Default Value": "デフォルト値",
```

File: extras/template/strings/shell-strings.json [hunk 5]
```diff
@@ -23,6 +23,13 @@
...
+        "Authors": "Authors",
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Authors": "作者",
```

File: extras/template/strings/shell-strings.json [hunk 6]
```diff
@@ -23,6 +23,13 @@
...
+        "Example": "Example",
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Example": "例",
```

File: extras/template/strings/shell-strings.json [hunk 7]
```diff
@@ -23,6 +23,13 @@
...
+        "Used by": "Used by"
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Used by": "次で使用:"
```

**Comment by Aleksandyr** [comment]
@valadzhov Could you please translate also this
        "See": "See",
        "Requires": "Requires"

**Comment by Aleksandyr** [comment]
@zdrawku and @HristoP96  The generation of SASSDOC jsons should be tested also!

File: extras/template/strings/shell-strings.json [hunk 8]
```diff
@@ -23,6 +23,15 @@
...
+        "Get Started": "はじめに",
+        "Name": "名",
+        "Description": "説明",
+        "Type": "型",
+        "Default Value": "デフォルト値",
+        "Authors": "作者",
+        "Example": "例",
+        "Used by": "次で使用",
+        "See": "See",
+        "Requires": "Requires"
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "Requires": "必須"
```

File: extras/template/strings/shell-strings.json [hunk 9]
```diff
@@ -23,6 +23,15 @@
...
-        "Get Started": "はじめに"
+        "Get Started": "はじめに",
+        "Name": "名",
+        "Description": "説明",
+        "Type": "型",
+        "Default Value": "デフォルト値",
+        "Authors": "作者",
+        "Example": "例",
+        "Used by": "次で使用",
+        "See": "See",
```
Comment (valadzhov) [inline extras/template/strings/shell-strings.json]:
```suggestion
        "See": "次を参照してください:",
```

**Pull request merged**